pkg/task/task.go

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
@obourdon you can checkout my branch and cherry-pick the commit to your PR.

On Mon, Jun 13, 2016 at 8:46 AM Olivier Bourdon <notifications@github.com>
wrote:

> In pkg/task/task.go
> <https://github.com/intelsdi-x/snap/pull/951#discussion_r66816318>:
>
> > @@ -0,0 +1,197 @@
> > +/*
>
> Seems perfect to me. @jcooklin <https://github.com/jcooklin> Thanks for
> this work Joel
> Is there a way we can substitute your PR instead of mine for this to get
> integrated ?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/intelsdi-x/snap/pull/951/files/531bdadfe945a51749dbce9976ea7fa1273ea3ea#r66816318>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA0q-J6U42k_7D2487MngUgqWbvllK3Aks5qLXtngaJpZM4Im5Oj>
> .
>